nixos/modules/services/x11/xautolock.nix

```diff
@@ -45,28 +55,80 @@ in
         };
 
         notifier = mkOption {
-          default = "notify-send 'Locking in 10 seconds'";
-          type = types.string;
+          default = null;
```

Comment:
as you added an assertion to ensure that `notifier` contains a useful script, `default = null` is not a good idea, right?

Reply:
Ah, I did this so that you'd have to take care of the script when you set `enableNotifier` to true, because the previous default `notify-send` was not mentioned in the manpage and would add another dependency.